astropy/units/tests/test_format.py

```diff
@@ -123,6 +124,8 @@ def test_unit_grammar_fail(string):
         (["[cm/s2]"], dex(u.cm / u.s**2)),
         (["[K]"], dex(u.K)),
         (["[-]"], dex(u.dimensionless_unscaled)),
+        (["eps0/mu0"], cds.eps0 / cds.mu0),
+        (["a0.s"], cds.a0 * u.s),
```

Comment:
No harm in adding a comment here saying that these two are regression tests for #17010.

Reply:
Not strictly true, just found it while investigating that. But more relevantly, this either means running all of CI again or an extra commit with `ci skip` - neither seems quite worth it to me, but happy to be told otherwise.

Reply:
I'm not a `units` maintainer, @nstarman should do the final review.

Reply:
I think in this case the git blame is good.
It would be good to find a way to move towards using Hypothesis, e.g. with these as [@example](https://hypothesis.readthedocs.io/en/latest/reproducing.html#hypothesis.example)

Reply:
We're using hypothesis on `Time`, where I found it rather painful and noisy (though obviously it does help correctness, but just felt that we ended up having to increase the tolerance slightly many times...).